### ISSUE TITLE
Support Sketch 45 plugin update system

### ISSUE THREAD

**Issue by bomberstudios** [andrewfiorillo/sketch-artboards-to-pdf#14]
Hi there!

This is a quick note to remind you that Sketch 45 will include a plugin update system.

It would be awesome if you could add support to it (it's really easy!) before Sketch 45 comes out. Ideally, you should release an update for your plugin while we're still in Sketch 44, so your users will have a nice & easy experience when 45 is released.

For more details, please check http://sketchplugins.com/d/229-updating-plugins and http://developer.sketchapp.com/introduction/updating-plugins/

Thanks in advance!

**Comment by slcr** [comment]
i don't think that is needed anymore, because with Sketch 45 you'll find this under "Share":
![bildschirmfoto 2017-07-05 um 14 49 45](https://user-images.githubusercontent.com/71357/27865029-371e0ee8-6191-11e7-94c6-f0b270d7c1d3.png)
